### PR TITLE
Add 13Clowns

### PR DESCRIPTION
**Pull request by 13Clowns**

### CODE REVIEW

File: default.py [hunk 1]
```diff
@@ -900,7 +903,7 @@ def maintMenu(view=None):
         addFile('--- Include Placenta: %s' % includeplacenta.replace('true',on).replace('false',off), 'togglecache', 'includeplacenta', icon=ICONMAINT, themeit=THEME3)
         addFile('--- Include Exodus Redux: %s' % includeexodusredux.replace('true',on).replace('false',off), 'togglecache', 'includeexodusredux', icon=ICONMAINT, themeit=THEME3)
         addFile('--- Include Gaia: %s' % includegaia.replace('true',on).replace('false',off), 'togglecache', 'includegaia', icon=ICONMAINT, themeit=THEME3)
-        addFile('--- Include Seren: %s' % includeseren.replace('true',on).replace('false',off), 'togglecache', 'includeseren', icon=ICONMAINT, themeit=THEME3)
+        addFile('--- Include 13Clowns: %s' % include13clowns.replace('true',on).replace('false',off), 'togglecache', 'include13clowns', icon=ICONMAINT, themeit=THEME3)addFile('--- Include Seren: %s' % includeseren.replace('true',on).replace('false',off), 'togglecache', 'includeseren', icon=ICONMAINT, themeit=THEME3)
```
Comment (drinfernoo) [inline default.py]: These lines appear that they'll be broken. Please fix them before I can merge this PR.

**Comment by drinfernoo** [comment]
I would also love a Zanni PR.

**Comment by 13Clowns** [comment]
fixed error on 906

zanni tomorrow!

**Comment by drinfernoo** [comment]
@13Clowns Do you want to submit it as a different PR or as part of this one?

**Comment by 13Clowns** [comment]
Wait for merge. Have a typo in Traktit.py

**Comment by drinfernoo** [comment]
You've got `HEAD` tags and `====`  in files. Either fix these, or I'll roll it in myself soon

**Comment by 13Clowns** [comment]
I see. I'll clean up in a few out shopping

**Comment by drinfernoo** [comment]
I took care of it.

**Comment by 13Clowns** [comment]
Thanks:smile:

**Pull request merged**